src/main/java/network/brightspots/rcv/CommonDataFormatReader.java

```diff
@@ -129,165 +129,168 @@ void parseXml(List<CastVoteRecord> castVoteRecords)
     // load XML
     XmlMapper xmlMapper = new XmlMapper();
     xmlMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-    FileInputStream inputStream = new FileInputStream(new File(filePath));
-    CastVoteRecordReport cvrReport = xmlMapper.readValue(inputStream, CastVoteRecordReport.class);
-
-    // Parse static election data:
-
-    // Find the Contest we are tabulating:
-    // Note: Contest is different from CVRContest objects which appear in CVRSnapshots
-    Contest contestToTabulate = null;
-    for (Election election : cvrReport.Election) {
-      for (Contest contest : election.Contest) {
-        if (contest.Name.equals(this.contestId)) {
-          contestToTabulate = contest;
-          break;
+    try (FileInputStream inputStream = new FileInputStream(filePath)) {
```

Comment:
Oh, interesting. I never knew about this feature (try with resources).

Reply:
It's a good one!